src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <article class="calendar calendar--start-day--sun calendar--month-length--29">
+      <p class="day day--1"></p>
```

Comment:
i'm pretty sure that `day` doesn't have to be a block, you won't reuse it outside of calendar anyway, so make it an element of calendar block